modules/nf-core/sourmash/compare/meta.yml

```
@@ -0,0 +1,63 @@
+name: "sourmash_compare"
+description: Compare many FracMinHash signatures generated by sourmash sketch.
+keywords:
+  - compare
+  - FracMinHash sketch
+  - containment
+  - sourmash
+  - metagenomics
+  - genomics
+  - kmer
+tools:
+  - "sourmash":
+      description: "Compute and compare FracMinHash signatures for DNA and protein data sets."
+      homepage: "https://sourmash.readthedocs.io/"
+      documentation: "https://sourmash.readthedocs.io/"
+      tool_dev_url: "https://github.com/sourmash-bio/sourmash"
+      doi: "10.21105/joss.00027"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - signatures:
+      type: file
+      description: Files containing signatures (hash sketches) of samples 
+      pattern: "*.{sig}"
+  - save_numpy_matrix:
+      type: boolean
+      description: |
+        If true, output will contain a (dis)similarity matrix numpy binary format
+  - save_csv:
+      type: boolean
+      description: |
+        If true, output will contain a (dis)similarity matrix in CSV format 
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - numpy_matrix:
+      type: file
+      description: A (dis)similarity matrix numpy binary format 
```

Is it similarity or dissimilarity (distance)?

```suggestion
      description: An optional (dis)similarity matrix numpy binary format 
```

---

by default it's a similarity matrix, but there's an optional command to make it a dissimilarity matrix